src/mol-model-props/computed/interactions/common.ts

```diff
@@ -158,7 +158,7 @@ export function interactionTypeLabel(type: InteractionType): string {
     }
 }
 
-export const enum FeatureType {
+export enum FeatureType {
```

Comment:
These are accessed fairly often, maybe worthwhile to use the `const enum FeatureType` and `enum FeatureTypes` pattern?